src/data/faq_de.json

```diff
@@ -800,72 +800,10 @@
-                                    "Wenn Ihr Zertifikat nicht mehr gültig ist oder Rechtschreibfehler/Zahlendreher enthält, wenden Sie sich an den Aussteller des Zertifikats, um ein neues Zertifikat zu erhalten. Bei anderweitigen Rückfragen wenden Sie sich an den Ticketanbieter."
-                                ]
-                            },
-                            {
-                                "title": "Warum ist mein Zertifikat nicht prüfbar oder wird nicht anerkannt?",
-                                "anchor": "val_service_result",
-                                "active": false,
-                                "textblock": [
-                                    "<b>Nicht anerkannt</b>",
-                                    "Wenn Sie als Ergebnis der Gültigkeitsprüfung die Meldung „Zertifikat nicht anerkannt“ erhalten, dann ist das Zertifikat technisch gültig aber entspricht nicht den Anforderungen des Anbieters.",
-                                    "<ul>",
-                                    "<li>Ein Antigentest ist älter als 48 Stunden und hat somit seine Gültigkeit verloren</li>",
-                                    "<li>Das Zertifikat wurde nicht von einem zertifizierten Testzentrum ausgestellt</li>",
-                                    "</ul>",
-                                    "<b>Zertifikat nicht prüfbar</b>",
-                                    "Wenn Sie als Ergebnis der Gültigkeitsprüfung die Meldung „Zertifikat nicht prüfbar“ erhalten, kann dies technische Gründe haben. Ihr Zertifikat ist in diesem Fall möglicherweise technisch veraltet oder es fehlen Daten innerhalb des Zertifikats, die zur Überprüfung benötigt werden.",
-                                    "Hier können Sie sich an den Aussteller Ihres Zertifikats wenden, um ein neues Zertifikat zu erhalten und den Ticketanbieter kontaktieren, wenn das Problem weiterhin bestehen sollte.",
-                                    "<b>Erfolgreiche Prüfung</b>",
-                                    "Das Zertifikat wurde erfolgreich überprüft und entspricht den Anforderungen des Anbieters."
-                                ]
+                                    "Unternehmen müssen gesetzliche Vorgaben zum Schutz der Gesundheit von Belegschaft und Kund*innen einhalten. Sie benötigen daher in der Pandemie ein verlässliches Zutrittsmanagement für beispielsweise Geschäftsräume, Werkshallen, Flugzeuge, Straßenbahnen, Hotels und Restaurants oder Veranstaltungen. Sie müssen schnell und unkompliziert die Echtheit von Digitalen COVID-Zertifikaten (DCCs) überprüfen können. Dem dient ein Corona Validation Service. Das Bundesministerium für Gesundheit (BMG) hat entschieden, mehrere Validation Services nach entsprechender Prüfung zu ermöglichen."

```

Comment:
Not currently correct:

"Das Bundesministerium für Gesundheit (BMG) hat entschieden, mehrere Validation Services nach entsprechender Prüfung zu ermöglichen."